### PR TITLE
Avatar custom size

### LINKED ISSUE #11256: Avatar support custom size
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Avatar can set custom size

### What does the proposed API look like?
```jsx
<Avatar size={64} />
```

### PR DESCRIPTION
**Pull request by emersonlaurentino**
closes #11256

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -215,242 +324,134 @@ Sean Sun <pinggodstudio@gmail.com>
...
 马斯特 <sd4399340@126.com>
-王集鹄 <wjhu111@21cn.com>
-柚子男 <yozman@sina.com>
-愚指导 <yutingzhao1991@sina.com>
-愚指导-TZ <yutingzhao1991@sina.com>
-超能刚哥 <margox@foxmail.com>
 马金花儿 <o.o@mug.dog>
-रोहन मल्होत्रा <rohan.malhotra@adwyze.com>
-白羊座小葛 <abeyuhang@gmail.com>
+黄子毅 <576625322@qq.com>
```
Comment (yesmeck) [inline AUTHORS.txt]: Could you revert this change? We will update it.

Comment (emersonlaurentino) [reply AUTHORS.txt]: Yes, of course. Should I remove myself, too?

Comment (yesmeck) [reply AUTHORS.txt]: You can keep yourself.

Comment (emersonlaurentino) [reply AUTHORS.txt]: done, AUTHORS.txt of feature-3.8.0 and I kept myself in it.

**Comment by emersonlaurentino** [comment]
I'm having this problem when I run `npm test`:

screenshot from 2018-07-31 12-19-18

A possible solution: https://github.com/facebook/jest/issues/6769#issuecomment-408352345
